src/deadline_test_fixtures/deadline/worker.py

```diff
@@ -595,28 +625,112 @@ def _start_worker_agent(self) -> None:
 
         LOG.info(f"Sending SSM command to start Worker agent on instance {self.instance_id}")
         cmd_result = self.send_command(
-            " && ".join(
+            "; ".join(
                 [
-                    f"nohup runuser --login {self.configuration.agent_user} -c 'AWS_DEFAULT_REGION={self.configuration.region} deadline-worker-agent > /tmp/worker-agent-stdout.txt 2>&1 &'",
-                    # Verify Worker is still running
-                    "echo Waiting 5s for agent to get started",
-                    "sleep 5",
-                    "echo 'Running pgrep to see if deadline-worker-agent is running'",
-                    f"pgrep --count --full -u {self.configuration.agent_user} deadline-worker-agent",
+                    " && ".join(
+                        [
+                            "set -x",
+                            f"nohup runuser --login {self.configuration.agent_user} -c 'AWS_DEFAULT_REGION={self.configuration.region} deadline-worker-agent > /tmp/worker-agent-stdout.txt 2>&1 &'",
+                            # Verify Worker is still running
+                            "echo Waiting 5s for agent to get started",
+                            "sleep 5",
+                            "echo 'Running pgrep to see if deadline-worker-agent is running'",
+                            # Note: pgrep has a non-zero exit code if no matching application can be found.
+                            f"pgrep --count --full -u {self.configuration.agent_user} deadline-worker-agent",
+                        ],
+                    ),
+                    # If the worker didn't start, then print out the agent logs that exist to aid in debugging.
+                    "if test $? -ne 0; then echo '+++AGENT NOT RUNNING+++'; cat /var/log/amazon/deadline/worker-agent-bootstrap.log /var/log/amazon/deadline/worker-agent.log; exit 1; fi",
```

Comment:
Added to make it much easier to diagnose issues from the logs when the agent fails to start.